fastflix/widgets/panels/status_panel.py

```diff
@@ -111,6 +121,40 @@ def update_bitrate(self, bitrate):
     def update_title_bar(self):
         pass
 
+    def set_started_at(self, msg):
+        try:
+            started_at = datetime.datetime.fromisoformat(msg.split("__")[-1])
+        except Exception as e:
+            logger.warning(f"Unable to parse start time - {str(e)}")
+            return
```

Comment:
Would also set started time to right now as a backup before returning

Reply:
Yes! Added in afd2d0b